dipy/core/gradients.py

```diff
@@ -244,3 +244,55 @@ def gradient_table(bvals, bvecs=None, big_delta=None, small_delta=None,
                                            small_delta=small_delta,
                                            b0_threshold=b0_threshold,
                                            atol=atol)
+
+
+def reorient_bvecs(gtab, rots):
```

Comment:
Where is the user expected to obtain the `rots` input from? For example, they may have the rotation matrices themselves instead of Euler angles